### ISSUE TITLE
[Bug]: Vertical TabList missing aria-orientation="vertical" attribute

### ISSUE THREAD

**Issue by adamsamec** [microsoft/fluentui#23964]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
N/A
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://react.fluentui.dev/?path=/docs/components-tablist--default

## Repro steps
1. Visit the [TabList component storybook documentation page ](https://react.fluentui.dev/?path=/docs/components-tablist--default).
2. Find the "Vertical" story of the TabList component.
3. Inspect the code of the tablist element.

## Actual Behavior
The element with the role="tablist" is missing the aria-orientation="vertical" attribute.

## Expected Behavior
The tablist element should have the aria-orientation="vertical" attribute set.

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by JustSlone** [comment]
tagging @smhigley and @GeoffCox
